test/utilities/outputView.ts

```diff
@@ -10,68 +10,35 @@ import { OutputView } from 'wdio-vscode-service';
 import { CMD_KEY } from 'wdio-vscode-service/dist/constants';
 import { pause } from './miscellaneous';
 import { dismissAllNotifications } from './notifications';
+import { runCommandFromCommandPrompt } from './commandPrompt';
 
-export async function selectOutputChannel(outputView: OutputView, name: string): Promise<void> {
+export async function selectOutputChannel(name: string): Promise<void> {
   // Wait for all notifications to go away.  If there is a notification that is overlapping and hiding the Output channel's
   // dropdown menu, calling select.click() doesn't work, so dismiss all notifications first before clicking the dropdown
   // menu and opening it.
   await dismissAllNotifications();
 
-  // Find the channel the Output view is current set to.
-  const dropDownMenu = await outputView.parent.$('select.monaco-select-box');
-  const currentChannelName = await dropDownMenu.getValue();
-  if (currentChannelName === name) {
-    // If the output channel is already set, don't do anything and just return.
-    return;
-  }
-
-  // Open the Output panel's dropdown menu.
-  await dropDownMenu.click();
-
-  // Click the target channel.
-  const channels = await dropDownMenu.$$('option');
-  for (const channel of channels) {
-    const val = await channel.getValue();
-    if (val === name) {
-      await channel.click();
-
-      // eslint-disable-next-line wdio/no-pause
-      await browser.pause(200);
-      await browser.keys(['Escape']);
-      await pause(1);
-      return;
-    }
-  }
-
-  throw new Error(`Channel ${name} not found`);
-}
-
-export async function openOutputView(): Promise<OutputView> {
+  // Find the given channel in the Output view
   const workbench = await (await browser.getWorkbench()).wait();
-  const bottomBar = await workbench.getBottomBar(); // selector is 'div[id="workbench.parts.panel"]'
-  const outputView = await bottomBar.openOutputView(); // selector is 'div[id="workbench.panel.output"]'
+  await runCommandFromCommandPrompt(workbench, 'Output: Show Output Channels...', 1);
+  await browser.keys([name, 'Enter']);
```

Comment:
This is a much neater solution than trying to find the right item from the dropdown. 😀